Motorcycle.js

```diff
@@ -1,6 +1,43 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
 
-module.exports = Motorcycle;
\ No newline at end of file
+// Part 1
+
+function Motorcycle(make, model, year, color, seats, passengers) {
+	Car.call(this, make, model, year, color, seats, passengers);
+	this.seats <= 2;
```

Comment:
Here, using a comparison operator won't achieve much. Instead, we'll need to use a conditional statement to override the value of seats when a user designates a number greater than 2:
```
if (this.seats > 2) {
		this.seats = 2;
	}
```